skimage/viewer/tests/test_tools.py

```diff
@@ -161,7 +161,7 @@ def test_rect_tool():
 @cleanup
 @testing.skipif(not has_qt, reason="Qt not installed")
 def test_paint_tool():
-    img = data.moon()
```

Comment:
could you parameterize this tests taht way we know it still works for gray images.

```python
from skimage._shared.testing import parametrize

@parametrize('img', [data.moon(), data.astronaut()])
def test_pain_tool(img):
```

Reply:
Sure no problem.

Reply:
cool thanks, and I think just for you qt pushed an update today that should break the tests ;)

Reply:
Is there a way to make Travis-CI rerun the tests?

Reply:
you gotta ask a core dev nicely ;)
I did it for you.
fixes for qt 5.12 just came in, so your tests will likely pass.

Optionally, I would amend a commit and force push. Force pushing is a bad habbit though.... but with the fork, branch, PR model, I think it is OK.

```
git commit --amend
git push --force
```

Reply:
I had to retrigger a build. I think we solved the particular issue on master, but travis didn't pick up the change. I simply removed a blank line, I hope that is ok.

Reply:
I think it will work now,
https://travis-ci.org/scikit-image/scikit-image/jobs/494273469#L159

the qt5dev stuff was missing at the end of that line before.

Reply:
That works...thanks.